Hyrax_DockerHub_Installation.adoc

```diff
@@ -0,0 +1,48 @@
+= Docker Installation (Recommended) =
+:Miguel Jimenez-Urias <mjimenez@opendap.org>:
+{docdate}
+:numbered:
+:toc:
+
+This is the simplest way to install and use the latest release of Hyrax. 
+
+== Prerequisites
+
+. `Docker` is running in the background.
+. You have a data folder. In this guide, we will assume it is `~/tmp/data/` .
+. Your data is stored in HDF5/NetCDF4 format, csv, or any other file format for which Hyrax has a data Handler (https://www.opendap.org/software/hyrax-data-server/[see all supported file formats by Hyrax])
+. OSX or Linux platform.
+
+== Run Hyrax and serve data
+
+. **Open a terminal and get the latest snapshot of hyrax**
+
+```
+docker pull opendap/hyrax:snapshot
+```
+
+TIP: Do not use `hyrax:latest`. We use Travis for CI/CD and `hyrax:snapshot` is fully tested, and packs with the correct versions needed to build Hyrax. Using `latest`, ironically may pull an older version of Hyrax.
```

Comment:
Rather than:
> Using `latest`, ironically may pull an older version of Hyrax. 

I think something like:
> The `latest` tag is associated with the "latest" official release of the Hyrax server. Official releases are labor intensive to produce and only happen once or twice a year. To get the most recent and up-to-date software, get the `snapshot` tag.

Reply:
👍🏽